src/data.ts

```diff
@@ -390,10 +399,18 @@ export function subRegionOneCode(region: RegionalTrendLine): string {
 
 export function subRegionTwoCode(region: RegionalTrendLine): string {
   if (region.sub_region_2_code != "") {
-    return region.sub_region_2_code
+    return region.sub_region_2_code;
+  } else if (["AU", "GB"].includes(region.country_region_code)) {
```

Comment:
Change this to:
} else {
  return getCountyFipsCode(region);
} 
and put all the country-specific logic in that method. it'll be a lot easier to follow :)